README.md

```diff
@@ -2,8 +2,8 @@
 
 [![npm version](https://badge.fury.io/js/blackbaud-stache.svg)](http://badge.fury.io/js/blackbaud-stache)
 
-## Building Stache
+## New Version of Stache
+
+We no longer support development on this version of Stache. Instead, we recommend you check out our <a href="https://github.com/blackbaud/stache2
+">latest version of Stache</a>, the Blackbaud Documentation Builder. The currently supported version is a collection of Angular 2 components for SKY UX SPAs. 
```

Comment:
I would say, "The currently support version is a collection of Angular components and SDK plugins for the SKY UX ecosystem."

Reply:
I guess I can make that change for you.